src/index.ts

```diff
@@ -61,109 +56,11 @@ export interface SpannerOptions extends GrpcClientOptions {
 export interface RequestConfig {
   client: string;
   method: string;
-  // tslint:disable-next-line: no-any
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
   reqOpts: any;
   gaxOpts?: {};
 }
-/*!
```

Comment:
uh ... won't that be bad for the generated docs?

Reply:
I don't know! Should be fine... probably.